WordPress/src/main/java/org/wordpress/android/util/DialogUtils.java

```diff
@@ -12,6 +12,9 @@
 import org.wordpress.android.widgets.WPTextView;
 
 public class DialogUtils {
+
+    private static OpenSansEditText editText;
```

Comment:
I think on rotation it will create a new instance of editText and it's value will not be saved. And how will callbacks will be more beneficial than using static?